src/Init/Data/Char/Lemmas.lean

```diff
@@ -22,13 +22,18 @@ protected theorem le_total (a b : Char) : a ≤ b ∨ b ≤ a := UInt32.le_total
 protected theorem lt_asymm {a b : Char} (h : a < b) : ¬ b < a := UInt32.lt_asymm h
 protected theorem ne_of_lt {a b : Char} (h : a < b) : a ≠ b := Char.ne_of_val_ne (UInt32.ne_of_lt h)
 
-theorem utf8Size_pos (c : Char) : 0 < c.utf8Size := by
-  simp only [utf8Size]
-  repeat (split; decide)
-  decide
+/-- `Char.size` now returns a `Nat`, rather than a `UInt32`. -/
+@[deprecated Char.size (since := "2024-06-04")] abbrev utf8Size := Char.size
```

Comment:
It seems like it would still be a good idea to return a `UInt32` from this for now to avoid breakages. The deprecation warning I think should be more than enough.

Reply:
Done.